config/controller.yaml

```diff
@@ -65,13 +65,16 @@ spec:
         app: kpack-controller
         version: #@ data.values.version
     spec:
+      priorityClassName: system-cluster-critical
```

Comment:
Where is the priority class `system-cluster-critical` defined?  I can only see `kpack-build-high-priority` and `kpack-build-low-priority` defined in `priority.yaml`.

Reply:
It's a default class. See https://kubernetes.io/docs/concepts/scheduling-eviction/pod-priority-preemption/#how-to-use-priority-and-preemption